pages/workshops_operations.html

```diff
@@ -481,7 +481,7 @@ <h3 id="host-registration">Registration</h3>
   the registration page further.
 </p>
 
-<h3 id="book-room">The Workshop Venue</h3>
+<h3 id="host_venue">The Workshop Venue</h3>
```

Comment:
~~~
$ grep -r book-room pages 
pages/workshops_operations.html:<h3 id="book-room">The Workshop Venue</h3>
$ grep -r host_venue pages
pages/checklists_host.html:      <a href="{{site.baseurl}}/workshops/operations/#host_venue">good venue</a> available?
pages/checklists_host.html:      the <a href="{{site.baseurl}}/workshops/operations/#host_venue">ideal room characteristics</a>.
~~~